website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+
+The overall framework of FLINT manages the processing of Simulation Units over time. Simulation Units are the basis of all simulations run in FLINT.
+
+### Synchronized events
+
+In some circumstances, modules that are simulated for a particular Simulation Unit are dependent on other Simulation Units. That is, there is an interdependency between the Simulation Units. In these situations it is necessary for there to be a process that synchronizes across the related Simulation Units and this is known as synchronized events. The Synchronized Server in FLINT manages Synchronized events.
+
+### Mass balance
+
+The carbon cycle is based on the transfer of carbon molecules from one pool to another, where the molecules are not created, nor are they destroyed. As such, all outputs must be equal to the inputs. The system must be in balance. This is referred to as mass balance. Overall this is the high level design of FLINT.
+
+|![Diagram depicting the high-level design of FLINT.](../static/img/flint-design-module-case-study.png) |
+|:--:|
+| Diagram depicting the high-level design of FLINT. |
+
+## Empirical Forest Growth Module
+
+The empirical Forest growth module includes all the operations required to simulate the biomass accumulation. Specifically, it simulates the growth in carbon stock within a forest and its components through time. It is a standard module used in FLINT and is used within Chapman Richards Module. **Biomass** refers to the mass of living organisms, including plants, animals and microorganisms. **Biomass accumulation** is the net change in standing biomass from one time to another (g/m^2).
+
+It runs at the **Monthly Step Interval**. It is based on the yield tables that present the **monthly growth increments** for a forest type. This data comes from research and experimentation and is called **empirical data**, thus giving the name of this module as empirical forest growth module. Yield Tables only include the **aboveground biomass**.
+
+### Inputs:
+
+-   Time Step (Monthly)
+-   Species
+-   Age
+-   Region
+-   Forest type
+-   Pool 1: Stemwood
+-   Pool 2: Bark
+-   Pool 3: Leaves
+-   Pool 4: Coarse Root
+-   Pool 5: Fine Roots
+-   Rainfall in that region
+-   Minimum and Maximum Temperature
+-   [Vapour Pressure Deficit](https://en.wikipedia.org/wiki/Vapour-pressure_deficit)
+-   Soil Water Holding Capacity
+
+### Outputs:
+
+-   Calculated aboveground and belowground Biomass.
+
+## Hybrid Forest Growth Module
+
+The Hybrid Forest Growth Module calculates the biomass above and below the ground. These modules use physiological responses to environmental conditions to derive relationships that can be used to predict tree growth. These modules contain equations or series of equations to calculate biomass.
```

Comment:
This is a bit confusing - is the Hybrid Forest Growth module one module or several?